doc/cascadia/profiles.schema.json

```diff
@@ -52,6 +52,117 @@
         }
       ]
     },
+    "AppearanceConfig": {
+      "properties": {
+        "colorScheme": {
+          "description": "The name of a color scheme to use when unfocused.",
+          "type": "string"
+        },
+        "foreground": {
+          "$ref": "#/definitions/Color",
+          "default": "#cccccc",
+          "description": "Sets the text color when unfocused. Overrides \"foreground\" from the color scheme. Uses hex color format: \"#rrggbb\".",
+          "type": ["string", "null"]
+        },
```

Comment:
i suspect there's a way to reuse this as a base for the profile settings so that we don't need two full copies of it! :smile:

Reply:
As discussed offline, we will deal with this in a follow up!